includes/handler.inc

```diff
@@ -458,6 +458,8 @@ function islandora_oai_query_solr($start = 0, $field = 'PID', $query = NULL, $se
     $query = $field . ':' . islandora_solr_lesser_escape($query);
   }
   $query_processor->buildQuery($query);
+  // Remove any base search filters.
+  $query_processor->solrParams = islandora_solr_remove_base_filters($query_processor->solrParams);
```

Comment:
I'm ok with the toggle just in this case, we have the precedent of doing so in the newspaper sp module,

```PHP
   if (variable_get('islandora_newspaper_solr_remove_base_filters', 1)) {
      module_load_include('inc', 'islandora_solr', 'includes/utilities');
      $solr_build->solrParams = islandora_solr_remove_base_filters($solr_build->solrParams);
    }
````

here
https://github.com/Islandora/islandora_solution_pack_newspaper/blob/52a33ada0ad5e3a0c1bb14a4be1852099d95d987/includes/utilities.inc#L221

But also, this has an dependency/requiremet issue ,  `module_load_include('inc', 'islandora_solr', 'includes/utilities'); ` is only loaded if query is not empty, which will yield a warning  when trying to call `islandora_solr_remove_base_filters` in that case.